hivemq4/k8s-image/config.xml

```diff
@@ -26,13 +26,10 @@
             <tcp>
                 <bind-address>${HIVEMQ_BIND_ADDRESS}</bind-address>
                 <bind-port>${HIVEMQ_CLUSTER_PORT}</bind-port>
-                <enabled>true</enabled>
             </tcp>
         </transport>
         <discovery>
-            <extension>
```

Comment:
Why did you remove this?

Reply:
The tag is still present. I just removed the reload interval as it prevents the hivemq startup on 4.10.

Reply:
`reload-interval` never did anything, it is an invalid tag, so can be removed safely